### ISSUE TITLE
Tab tree sidebar completely blank

### ISSUE THREAD

**Issue by avamk** [traxium/tabtree#72]
Hello,

I installed the latest Tab Tree via the Firefox 43.0.1 Add-ons Manager, and the sidebar with the tabs is completely blank/white. The tabs are actually there because I can click on them, but since I can't see anything that's a big problem? Can anyone help with this? Thanks.

**Comment by traxium** [comment]
Yes. See #12 #53. Probably you have Intel graphics. Disabling "Use hardware acceleration when available" in Firefox options (Options -> Advanced -> General) should help.

**Comment by avamk** [comment]
Hello,

I just tried that, unfortunately the problem persists...

**Comment by traxium** [comment]
Just to be sure. Do you have Intel graphics card?

**Comment by avamk** [comment]
Yes I have a fairly old Intel GMA 4500MHD.

**Comment by ToLive** [comment]
I've "solved" this issue by activating "layers.d3d11.force-warp" to "true"
